src/test/kotlin/no/nav/syfo/brev/narmesteleder/NarmesteLederAccessServiceSpek.kt

```diff
@@ -0,0 +1,282 @@
+            val dialogmotedeltakerArbeidstaker = generateDialogmotedeltakerArbeidstaker()
+            val dialogmoteList = listOf(generateDialogmote())
+
+            val dialogmotedeltakerServiceMock = mockk<DialogmotedeltakerService>()
+            val narmesteLederClientMock = mockk<NarmesteLederClient>()
+            val oppfolgingstilfelleClientMock = mockk<OppfolgingstilfelleClient>()
+            val referatCreated6MonthsPrior = generateReferat(6L)
+            val referatCreated3MonthsPrior = generateReferat(3L)
+
+            val narmesteLederAccessService = NarmesteLederAccessService(
+                dialogmotedeltakerServiceMock,
+                narmesteLederClientMock,
+                oppfolgingstilfelleClientMock
+            )
+
+            beforeEachTest {
+                clearMocks(dialogmotedeltakerServiceMock)
+                clearMocks(narmesteLederClientMock)
+                clearMocks(oppfolgingstilfelleClientMock)
+
+                coEvery {
+                    dialogmotedeltakerServiceMock.getDialogmoteDeltakerArbeidsgiverById(
+                        any(),
+                    )
+                } returns dialogmotedeltakerArbeidsgiver
+
+                coEvery {
+                    dialogmotedeltakerServiceMock.getDialogmoteDeltakerArbeidstaker(
+                        any(),
+                    )
+                } returns dialogmotedeltakerArbeidstaker
+            }
+
+            describe("filterExpiredDocumentsInDialogmoter") {
+                it("should return brev within overlapping oppfolgingstilfelle start date") {
+                    coEvery {
+                        oppfolgingstilfelleClientMock.oppfolgingstilfelleTokenx(
+                            any(),
+                            any(),
+                            any(),
+                            any(),
+                            any()
+                        )
+                    } returns listOf(
+                        Oppfolgingstilfelle(
+                            start = LocalDate.now().minusMonths(6),
+                            end = LocalDate.now().minusMonths(2)
+                        )
+                    )
+
+                    val moteList = runBlocking {
+                        narmesteLederAccessService.filterExpiredDocumentsInDialogmoter(
+                            moteList = dialogmoteList,
+                            narmesteLederPersonIdentNumber = UserConstants.NARMESTELEDER_FNR,
+                            arbeidstakerPersonIdentNumber = UserConstants.ARBEIDSTAKER_FNR,
+                            tokenx = anyToken,
+                            callId = anyCallId
+                        )
+                    }
+                    val brev = moteList.toNarmesteLederBrevDTOList()
+
+                    brev.size shouldBe 4
+                }
+
+                it("should only return brev created within 4 months when oppfolgingstilfelle is past 4 months") {
```

Comment:
Pirker litt mer jeg.. Mener du med _past_, at tilfellet er lengre enn 4 måneder? Kanskje det kan byttes ut med _is more than 4 months long_ elns?

Reply:
Det jeg prøver å si er at oppfolgingstilfelle har en sluttdato mer enn 4 måneder siden, at den ikke overlapper den 4 månedersgrensen. Høres `should only return brev created within 4 months when returned oppfolgingstilfelle ended more than 4 months ago`?